doc/swss-schema.md

```diff
@@ -630,6 +634,35 @@ Equivalent RedisDB entry:
 
 ## Configuration DB schema
 
+### PORT_TABLE
+Stores information for physical switch ports managed by the switch chip. Ports to the CPU (ie: management port) and logical ports (loopback) are not declared in the PORT_TABLE. See MGMT_PORT.
+
+    ;Configuration for layer 2 ports
+    port_table_key      = PORT|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    admin_status        = BIT           ; is the port enabled (1) or disabled (0)
+    lanes               = list of lanes ; (need format spec???)
+    ifname              = 1*64VCHAR     ; name of the port, must be unique
+    mac                 = 12HEXDIG      ;
+    alias               = 1*64VCHAR     ; alias name of the port used by LLDP and SNMP, must be unique
+    description         = 1*64VCHAR     ; port description
+    speed               = 1*6DIGIT      ; port line speed in Mbps
+    mtu                 = 1*4DIGIT      ; port MTU
+    fec                 = 1*64VCHAR     ; port fec mode
+    autoneg             = BIT           ; auto-negotiation mode
+
+### MGMT_PORT_TABLE
+    ;Configuration for management port, including at least one key
+    port_table_key      = MGMT_PORT|ifname    ; ifname must be unique across PORT,INTF,VLAN,LAG TABLES
+    admin_status        = BIT           ; is the port enabled (1) or disabled (0)
+    ifname              = 1*64VCHAR     ; name of the port, must be unique, eg. eth0
+    mac                 = 12HEXDIG      ;
+    alias               = 1*64VCHAR     ; alias name of the port used by LLDP and SNMP, must be unique
+    description         = 1*64VCHAR     ; port description
+    speed               = 1*6DIGIT      ; port line speed in Mbps
+    mtu                 = 1*4DIGIT      ; port MTU
+    fec                 = 1*64VCHAR     ; port fec mode
+    autoneg             = BIT           ; auto-negotiation mode
```

Comment:
do we need these configurations for the management port? e.g. ifname, fec, autoneg?

Reply:
fec and autoneg is optional. Let's design it first.